src/config.rs

```diff
@@ -31,6 +46,7 @@ impl<'a> Default for Config<&'a str> {
             // :^)
             window_title: "Super Mini GL Framebufferer 3!",
             window_size: (600.0, 480.0),
+            invert_y: true

```

Comment:
> I would probably have more credibility to argue this if I didn't hard cod a silly window title, but invert_y: true feels very opinionated for Default 😄

I'd say it's actually more opinionated on my part, since I'm saying that the buffer starting at the bottom of the window is inverted from the way it should be. :P 

Most people expect libraries that have to do with pixels to have an origin in the top left. The fact that this is the default is just how it used to be. We don't need to make TOO many breaking changes now, do we? :P